string marker_name
string owner